packages/sdk/src/oracle/client.ts

```diff
@@ -132,12 +137,15 @@ export class Client {
     );
   }
   // runs statemachine step loop pretty fast by default.
-  startInterval(delayMs = 10): void {
+  startInterval(delayMs = 1): void {
     assert(!this.intervalStarted, "Interval already started, try stopping first");
     this.intervalStarted = true;
     loop(async () => {
       assert(this.intervalStarted, "Interval Stopped");
+      // it turns out since these 2 state machines share the same immer state, they need to be run serially and
+      // cant be run concurrently or you get wierd state oscillations. For now keep them in the same timing loop.
       await this.sm.tick();
+      await this.poller.tick();
```

Comment:
state machines are in charge writes to global state, and there was a bug due to inconsistent state across concurrent writes, so this had to be moved in series to prevent the issue. bug unrelated to pr.